doc/flakes/design.md

```diff
@@ -101,23 +89,26 @@ module.
   # Some other metadata.
   description = "A filesystem that fetches DWARF debug info from the Internet on demand";
 
-  # A list of flake references denoting the flakes that this flake
-  # depends on. Nix will resolve and fetch these flakes and pass them
-  # as a function argument to `outputs` below.
+  # The flake dependencies. Nix will resolve and fetch these flakes and pass
+  # them as a function argument to `outputs` below.
   #
-  # `flake:nixpkgs` denotes a flake named `nixpkgs` which is looked up
+  # "nixpkgs" denotes a flake named `nixpkgs` which is looked up
   # in the flake registry, or in `flake.lock` inside this flake, if it
   # exists.
   inputs = [ flake:nixpkgs ];
```

Comment:
This probably should be changed to `"nixpkgs"` (though `flake:nixpkgs` also works).